src/org/sosy_lab/cpachecker/cfa/ast/c/CIntegerLiteralExpression.java

```diff
@@ -21,6 +21,7 @@ public class CIntegerLiteralExpression extends AIntegerLiteralExpression
   private static final long serialVersionUID = 7691279268370356228L;
   public static final CIntegerLiteralExpression ONE = createDummyLiteral(1L, CNumericTypes.INT);
   public static final CIntegerLiteralExpression ZERO = createDummyLiteral(0L, CNumericTypes.INT);
+  public static final CIntegerLiteralExpression THREE = createDummyLiteral(3L, CNumericTypes.INT);
```

Comment:
Please move test value to corresponding CTypeToStringAlignmentTest

Reply:
Done